main.lua

```diff
@@ -1366,3 +1350,71 @@ function userdataIterator(data)
         if success then return idx, item end
     end
 end
+
+local LSPCmds = {
```

Comment:
nit: the naming convention I'm using is to reserve TitleCased names for class-like objects, use ALL_UPPERCASE for constants and camelCase for variables / functions.

Reply:
I used something similar to your LSP* names. I will use uppercase since it "is" a constant table.